db/models.py

```diff
@@ -50,3 +57,60 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
The `Order` model references the `User` model directly. It's recommended to use `settings.AUTH_USER_MODEL` to ensure compatibility with custom user models.